FFEmployee+CoreDataProperties.h

```diff
@@ -0,0 +1,26 @@
+//
+//  FFEmployee+CoreDataProperties.h
+//  TutorialApplication
+//
+//  Created by Александр Егоров on 1/31/17.
+//  Copyright © 2017 Александр Егоров. All rights reserved.
+//
+
+#import "FFEmployee+CoreDataClass.h"
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FFEmployee (CoreDataProperties)
+
++ (NSFetchRequest<FFEmployee *> *)fetchRequest;
+
+@property (nullable, nonatomic, copy) NSString *firstName;
+@property (nullable, nonatomic, copy) NSString *fullName;
```

Comment:
Full name should not be stored in the database, it should be calculated dynamically.